examples/lakefs/enterprise/values-external-aws.yaml

```
@@ -0,0 +1,71 @@
+lakefsConfig: |
+  logging:
+      level: "INFO"
+  blockstore:
+    type: local
+  auth:
+    authentication_api:
+      endpoint: http://localhost:8080/api/v1
+      external_principals_enabled: true
+    api:
+      endpoint: http://localhost:9006/api/v1
+    remote_authenticator:
+      enabled: true
+      endpoint: http://localhost:9006/api/v1
+    ui_config:
+      login_cookie_names:
+        - internal_auth_session
+ingress:
+  enabled: true
+  ingressClassName: <class-name>
+  hosts:
+    # the ingress that will be created for lakeFS
+    - host: <lakefs.ingress.domain>
+      paths:
+        - /
+
+##################################################
+########### lakeFS enterprise - FLUFFY ###########
+##################################################
+
+fluffy:
+  enabled: true
+  image:
+    repository: treeverse/fluffy
+    tag: '0.3.0'
+    pullPolicy: IfNotPresent
+    privateRegistry:
+      enabled: true
+      secretToken: <dockerhub-token-fluffy-image>
+  fluffyConfig: |
+    logging:
+      format: "json"
+      level: "INFO"
+    auth:
+      serve_disable_authentication: true
+      serve_listen_address: localhost:9006
+      encrypt:
+        secret_key: shared-secrey-key
+      external:
+        aws_auth:
+          enabled: true
+          get_caller_identity_max_age: 60
+    
+          # list of headers that are required to be present in the GetCallerIdentity request
+          required_headers:
+            required-key: "custom-value"
```

I wonder maybe it's better to use the ingress host provided to add `X-LakeFS-Server-ID`  ? it will save us a lot of typos and troubleshooting with clients. so by default configure `required_headers` with `X-LakeFS-Server-ID: <lakefs.ingress.domain>` using helpers and environment variables in the chart. Only if the client configures here in the values `required_headers` it will override it! Thoughts? 

```suggestion
          # required_headers:
            # required-key: "custom-value"
            # this header added by default in all clients to make sure one does authenticate with staging server and then reuses the token in production, so each request should be done per host
            # X-LakeFS-Server-ID: <lakefs.ingress.domain>
```